kadai1/takkyuuplayer/imgconv/imgconv.go

```diff
@@ -0,0 +1,127 @@
+package imgconv
+
+import (
+	"fmt"
+	"image"
+	"image/gif"
+	"image/jpeg"
+	"image/png"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// Converter is a image converter
+type Converter struct {
+	Directory string
+	// The extension to convert images from
+	FromExt string
+	// The extension to convert images to
+	ToExt string
+}
+
+var supported = map[string]bool{
```

Comment:
```suggestion
var supported = map[string]struct{}{
```
valueを使わないのであれば、空のstructは0バイトなのでおすすめです

Reply:
それはいいですね！変えます 💨